frontend/src/components/ProjectsAnalytics/index.js

```diff
@@ -0,0 +1,103 @@
+import "./index.css";
+import X from "../../assets/X.png";
+import { SET_ACTIVE_PROJECT_ANALYTICS } from "../../redux/actions/billingActions";
+import { useDispatch, useSelector } from "react-redux";
+import Chart from "react-apexcharts";
+import { appColor } from "../../constants";
+
+function ProjectsAnalytics() {
+  const dispatch = useDispatch();
+  const dataSource = useSelector(
+    (state) => state.billingReducer.billingList
+  ).filter((item) => item.cost > 0);
+  const projectsCostsSum = dataSource.reduce((acc, { fk_project_id, cost }) => {
+    if (acc[fk_project_id]) {
+      acc[fk_project_id] += cost;
+    } else {
+      if (cost != 0) {
+        acc[fk_project_id] = cost;
+      }
+    }
+    return acc;
+  }, {});
+  const uniqueProjects = Object.entries(projectsCostsSum)
+    .map(([fk_project_id, cost]) => ({ project: fk_project_id, cost }))
+    .sort((a, b) => b.cost - a.cost);
+  const categories = uniqueProjects.map(project => project.project);
+  const costData = uniqueProjects.map(project => project.cost);
+
+  var projectsChart = {
+      series: costData,
+      options: {
+        colors: [ "#5976E1", "#4E3340", "#CF6973", "#D4D145", "#ECCC4C",],
```

Comment:
Let's put this into the appColor constant file, so we can easily match styling if we need to